crates/rspack_loader_runner/src/scheme.rs

```diff
@@ -114,7 +118,12 @@ pub fn get_scheme(specifier: &str) -> Scheme {
     }
   }
 
-  Scheme::from(specifier[..i].to_ascii_lowercase().as_str())
+  let scheme_str = &specifier[..i].to_ascii_lowercase();
+  if scheme_str == "http" || scheme_str == "https" {
```

Comment:
I think this would be better to move to `From<&str>` implementation for `Scheme`.

Reply:
I'll take a look. Still learning rust haha

Reply:
I refactored a little bit, this is included